tests/Unit/ParallelAlgorithms/Actions/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY "Test_Actions")
```

Comment:
This seems a bit too generic for a globally unique identifier.  How about something like "Test_ParallelAlgorithmsActions"?

Reply:
Yes, that is consistent with how the libraries are named for other generic subdirectories

Reply:
that's better, thanks for the suggestion